curp/tests/common/curp_group.rs

```diff
@@ -107,4 +126,92 @@ impl CurpGroup {
     ) -> impl Iterator<Item = &mut mpsc::UnboundedReceiver<(TestCommand, LogIndex)>> {
         self.nodes.values_mut().map(|node| &mut node.as_rx)
     }
+
+    pub fn stop(mut self) {
+        thread::spawn(move || {
+            self.nodes.clear();
+        })
+        .join()
```

Comment:
can we call `self.nodes.clear()` directly, who spawn a thread and join here

Reply:
Sry, nope. Because every node corresponds to a tokio runtime, which cannot be destroyed in another tokio runtime.